Servers/Administration/OJS.Servers.Administration/Properties/launchSettings.json

```diff
@@ -25,8 +25,8 @@
     "OJS.Servers.Administration": {
       "commandName": "Project",
       "dotnetRunMessages": "true",
-      "launchBrowser": true,
-      "applicationUrl": "https://localhost:5001",
+      "launchBrowser": false,
+      "applicationUrl": "http://localhost:5001",
```

Comment:
Why remove the https?

Reply:
Applications should run on http. It's the role of the web server, i.e. NGINX, to verify the SSL